git-integration/README.md

```diff
@@ -0,0 +1,8 @@
+# Roadmap for Git Integration
+
+Many notebook users utilize git and GitHub to manage versions of
+their notebooks and share them with others. Sometimes this process
+can be quite difficult as creating diffs of the notebook format
+comes with a lot of challenges. We are hoping to address this
+issue and create tightly-coupled git integration in the nteract
+desktop application.
```

Comment:
You could add some things for the opinionated stance of storing notebooks in lfs, or enabling UI for actions that some people have requested, e.g. strip-on-commit. Hopefully the nbdime diff rendering should be usable in an nteract context once it's more polished.